lib/interactiveui.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+const execa = require('execa');
+const inquirer = require('inquirer');
+const readPkgUp = require('read-pkg-up');
+const version = require('./version');
+
+module.exports = options => {
+	const pkg = readPkgUp.sync().pkg;
+
+	const prompts = [
+		{
+			type: 'list',
+			name: 'version',
+			message: 'Select semver increment or specify new version',
+			choices: version.SEMVER_INCREMENTS.concat([
```

Comment:
I think we should show what the version would end up being with each increment:

```
patch (2.3.1)
minor (2.4.0)
major (3.0.0)
```

And the `(2.3.1)` should be dimmed with `chalk.gray.dim(text)`.

https://github.com/chalk/chalk